blimflix/client/app/seen/seen.controller.js

```diff
@@ -1,55 +1,28 @@
 (function() {
   'use strict';
 
-  /**
-   * Chain to fetch module
-   *   https://github.com/johnpapa/angular-styleguide#style-y022
-   */
   angular
     .module('blimFlix.seen')
     .controller('SeenController', SeenController);
     SeenController.$inject = ['seen'];
 
-  /**
-   * Avoid anonymous functions as callbacks
-   *   https://github.com/johnpapa/angular-styleguide#style-y024
-   *
-   * Document dependency injenction using annotations
-   *  https://github.com/johnpapa/angular-styleguide#style-y100
-   */
-  /* @ngInject */
   function SeenController(seen) {
-    // var movies = seen.get({id:1});
-    // console.log(movies);
-    /**
-     * Attach any view properties to this variable
-     *   https://github.com/johnpapa/angular-styleguide#style-y032
-     */
-    /* jshint validthis: true */
+
     var vm = this;
+    vm.seenMoviesArray = [];
+    vm.seenMovies = [];
 
-    /**
-     * Bindable vm.* members (in alphabetical order)
-     *   All variables here should be part of the vm object and not bound to `this`
-     *   https://github.com/johnpapa/angular-styleguide#style-y033
-     */
-    vm.activate = activate;
+    //Get for seen movies
+    seen.query({ id: 1 }).$promise.then(function (data) {
```

Comment:
It should use the user object in the main controller, If it is not created, we can make a singleton object to get the user data at any place of the application